crypto/keyring/keyring.go

```diff
@@ -882,22 +883,31 @@ func (ks keystore) migrate(version uint32, i keyring.Item) error {
 		if len(item.Data) == 0 {
 			return sdkerrors.Wrap(sdkerrors.ErrKeyNotFound, key)
 		}
-
-		info, err := aminoUnmarshalInfo(item.Data)
+	
+		info, err := protoUnmarshalInfo(item.Data)
 		if err != nil {
 			return err
 		}
-		// TODO:
-		// + marshal as proto
-		// + ks.db.Set(item keyring.Item)
-	}
+
+		var versionBytes = make([]byte, 4)
+		binary.LittleEndian.PutUint32(versionBytes, CURRENT_VERSION)
+		ks.db.Set(keyring.Item{
+			Key:         info.GetKey().String(),
```

Comment:
should we set info.public Key to Key field of keyring.Item here?

Reply:
* `Key`, is the key name - the `key` variable set in the loop.
* `Data` is proto serialized `Info`